src/util/createServerRootMixin.js

```diff
@@ -104,6 +104,11 @@ function augmentInstantSearch(instantSearchOptions, cloneComponent) {
         app = cloneComponent(component, {
           mixins: [
             {
+              beforeCreate() {
+                // In case of Nuxt (3), we ensure the context is shared
+                // Note that this has no effect outside of Nuxt
+                this.$nuxt = component.$nuxt;
```

Comment:
Should we wrap this in a condition to check whether `component.$nuxt` actually exists?

Reply:
yes, good point